packages/smooth_app/lib/pages/product/edit_product_page.dart

```diff
@@ -118,175 +118,179 @@ class _EditProductPageState extends State<EditProductPage> {
           barcode: _barcode,
           widget: this,
         ),
-        child: Scrollbar(
-          child: ListView(
-            controller: _controller,
-            children: <Widget>[
-              if (_ProductBarcode.isAValidBarcode(_product.barcode))
-                _ProductBarcode(product: _product),
-              _ListTitleItem(
-                title: appLocalizations.edit_product_form_item_details_title,
-                subtitle:
-                    appLocalizations.edit_product_form_item_details_subtitle,
-                onTap: () async {
-                  if (!await ProductRefresher().checkIfLoggedIn(context)) {
-                    return;
-                  }
-
-                  AnalyticsHelper.trackProductEdit(
-                      AnalyticsEditEvents.basicDetails, _barcode);
-
-                  await Navigator.push<void>(
-                    context,
-                    MaterialPageRoute<void>(
-                      builder: (_) => AddBasicDetailsPage(_product),
-                      fullscreenDialog: true,
-                    ),
-                  );
-                },
-              ),
-              _ListTitleItem(
-                leading: const Icon(Icons.add_a_photo_rounded),
-                title: appLocalizations.edit_product_form_item_photos_title,
-                subtitle:
-                    appLocalizations.edit_product_form_item_photos_subtitle,
-                onTap: () async {
-                  AnalyticsHelper.trackProductEdit(
-                      AnalyticsEditEvents.photos, _barcode);
-
-                  await Navigator.push<void>(
-                    context,
-                    MaterialPageRoute<void>(
-                      builder: (BuildContext context) =>
-                          ProductImageGalleryView(
-                        product: _product,
+        child: PrimaryScrollController(
+          controller: _controller,
+          child: Scrollbar(
+            child: ListView(
```

Comment:
```suggestion
        child: ListView(
          controller: _controller,
```

Just curious: could you try this instead? That would look much easier.

Reply:
> Just curious: could you try this instead? That would look much easier.

@g123k ping
It should be a quick test and if it's not successful I'll quickly approve the current PR.

Reply:
Oops didn't saw your comment.
Basically it will do the same, by inheriting from it, it think it's better as it allows to use a different one in the future

Reply:
I really have a preference for the simple syntax if it does exactly the same.
And I would let the future developers worry about using a different syntax only if they need it.